src/basic/missing_input.h

```diff
@@ -11,9 +12,9 @@
 /* linux@06a16293f71927f756dcf37558a79c0b05a91641 (4.4) */
 #ifndef EVIOCSMASK
 struct input_mask {
-        uint32_t type;
-        uint32_t codes_size;
-        uint64_t codes_ptr;
+        __u32 type;
+        __u32 codes_size;
+        __u64 codes_ptr;
```

Comment:
What is the advantage of this?

Reply:
I guess this has no big advantage. I just want to keep the current kernel definitions. Also, by using this way, we do not need any additional headers to implement each `missing_*.h` except `linux/types.h`.

Reply:
That last thing is certainly a valid point.